Update dependency for guzzlehttp/guzzle to ^6.2|^7.0

cyrillkalita opened the pull request:
Enable support for Guzzle 7.0 #45

Commit:
Update dependency for guzzlehttp/guzzle to ^6.2|^7.0

ngunyimacharia commented:
Is there a way we can help to get this MR merged? All the checks are passing and an approving review has already been granted

ngunyimacharia committed:
Fix: Double pipe for logical OR

Documentation below for version ranges

https://getcomposer.org/doc/articles/versions.md#version-range

ngunyimacharia commented on composer.json:
```diff
@@ -25,7 +25,7 @@
     "require": {
         "php": "^7.1",
-        "guzzlehttp/guzzle": "^6.2",
+        "guzzlehttp/guzzle": "^6.2|^7.0",
```
Based on Composer's documentation, we should use a double pipe for logical OR with regards to version ranges. I have created a pull request for this.

ngunyimacharia committed:
Update illuminate/support dependency to ^8.0

cyrillkalita committed:
Merge pull request #1 from ngunyimacharia/patch-1

Fix: Double pipe for logical OR

cyrillkalita commented:
TinyURL test seems to be consistently failing

axlon commented:
@cyrillkalita Hey there, first of all thanks for the PR, however I doubt it will be merged, all active members of this organization have left it (including me, and I was the person primarily driving its development). Because I no longer have access to merge or tag this I would recommend you use your fork for now.

Pinging @yak0d3, I think it would be a good idea to transfer ownership of this repository to someone who can devote time to it or mark it as abandoned within packagist
